database/mssql/scripts/dbscripts/PSP_PIMS_S25_00/Alter Down/10_DML_PIMS_STATIC_VARIABLE_VERSION.sql

```diff
@@ -0,0 +1,26 @@
+/* -----------------------------------------------------------------------------
```

Comment:
This is what I mean, I would not expect a PR for 25.1 to have changes to the PSP_PIMS_S25_00 folder.

Reply:
Looks like it's correct and unchanged.  Not sure why this would be included in 25.01.  When I was tlaking with Manuel he was asking me about something and perhaps this was what he meant.

Reply:
Could be the way you are creating your branches. When creating new branches, from dev, always make sure your branch is up-to-date with bcgov/dev before creating the branch.

Reply:
I suspect this PR was created **before** the 25 database changes were merged into dev. So the git diff would include those as well

Reply:
Yes, I'm ready to blow this away and re-PR but waiting for the OK to do so.